packages/lab/src/components/Flow/DroppableFlow.tsx

```diff
@@ -39,6 +39,8 @@ export interface HvDroppableFlowProps<
   nodes?: Node<NodeData, NodeType>[];
   /** Flow edges. */
   edges?: Edge[];
+  /** Flow empty state. */
+  emptyState?: React.ReactNode;
```

Comment:
We no longer need the `emptyState` prop in the droppable flow (and all the related code added).